plugins/api-docs/src/components/ApiExplorerTable/presets.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  createApiEntityRefColumn,
+  createApiTagsColumn,
+  createApiTypeColumn,
+  createDescriptionColumn,
+  createLifecycleColumn,
+  createOwnerColumn,
+  createSystemColumn,
+} from './columns';
+import {
+  createTypeFilter,
+  createLifecycleFilter,
+  createTagsFilter,
+  createOwnerFilter,
+} from './filters';
+import { Columns, Filters } from './types';
+
+export const defaultColumns: Columns = {
+  name: createApiEntityRefColumn(),
+  system: createSystemColumn(),
+  owner: createOwnerColumn(),
+  lifecycle: createLifecycleColumn(),
+  type: createApiTypeColumn(),
+  description: createDescriptionColumn(),
+  tags: createApiTagsColumn(),
+};
```

Comment:
I borrowed this from the `EntityTable` presets. However, other components in Backstage follow the pattern of declaring functions for creating columns, which can later be composed as part of a list as seen [here](https://github.com/backstage/backstage/blob/8b3cf2838eca9c8c96797c14e793061e0fcfe2a6/plugins/catalog-react/src/components/EntityTable/presets.tsx#L29). The idea of using a `Record` type came from an earlier suggestion, but now I'm wondering if having created those functions was overkill or if I should be consistent with the pattern that other components follow.